finance.adoc

```diff
@@ -31,6 +31,7 @@ https://finance.izettle.com
 *Example responses:*
```

Comment:
as we have already used a header styling, a colon ":" is not needed.

Reply:
I changed this to just **Examples** after our meeting today.
Can you please review? 🙏

Reply:
Yes, it looks great with what you've changed. :D
I like your structure:
**Example** 
then **Request** 
and then **Response**